src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -509,6 +509,31 @@ public void testExcludeTags() throws Exception {
         assertMetric("test1.histogram", 424242, commonTags, 2, "histogram");
     }
 
+    @Test
+    public void testExcludeServiceTagsAndOverride() throws Exception {
+        // We expose a few metrics through JMX
+        SimpleTestJavaApp testApp = new SimpleTestJavaApp();
+        registerMBean(testApp, "org.datadog.jmxfetch.test:type=SimpleTestJavaApp,service=foo");
```

Comment:
Might be worth adding a test for the case when a service bean parameter is present, the `service` config field is configured, and the service tag isn't excluded: I think the expectation is that both the service bean parameter and the service config field are present as tags on the metrics. (I'm not sure how much that makes sense though, so maybe worth commenting that this is behavior that could be changed in the future)